ci/e2e-tests.sh

```diff
@@ -0,0 +1,615 @@
+#!/bin/bash
+
+# Usage:
+#
+# e2e-tests.sh <test_name>
+#
+# See /docs/dev/e2e-tests.md for details of some env vars that need to be defined.
```

Comment:
Should this link to the html page? I assume this page will be available with the rest of the docs.

Reply:
Done.